modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/MultiSelect.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+import ClayMultiSelect from '@clayui/multi-select';
+import React, {useState} from 'react';
+
+export default function MultiSelect({
```

Comment:
As I told I recommend you to add explicitly here ALL the props you know you can receive. You don't NEED to declare all of them, but it will help with the reading and to understand how this works. 

```
export default function MultiSelect({
	additionalProps: _additionalProps,
	clearAllTitle,
	closeButtonAriaLabel,
	componentId: _componentId,
	cssClass,
	disabled,
	disabledClearAll,
	inputName,
	inputValue: _inputValue,
	isValid,
	locale: _locale,
	locator,
	portletId: _portletId,
	portletNamespace: _portletNamespace,
	selectedItems: _selectedItems = [],
	sourceItems,
	...otherProps
}) {
```

Reply:
Good to know that it's not needed, but beneficial to have.